### ISSUE TITLE
the different modes don't read when arrowing through the combo box.bug: 

### ISSUE THREAD

**Issue by armstrong25** [jfayre/talking-flight-monitor-net#12]
when you tab to the combo box for the different mods such as fly plan or approach in the latest version of TFM, when you hit down arrow, NVDA does not read when you change modes.**Describe the bug**

**To Reproduce**

**Expected behavior**

**additional information
 - OS: [e.g. Windows 10]
 - TFM version: 
 - Flight simulator in use: 

**Additional context**
Add any other context about the problem here. What other add-ons were you running at the time, etc.
